internal/cmd/argus/grafana/describe/describe.go

```diff
@@ -0,0 +1,145 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+)
+
+const (
+	instanceIdFlag   = "instance-id"
+	showPasswordFlag = "show-password"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	InstanceId   string
+	ShowPassword bool
```

Comment:
In other commands we use the inverse, `hide-password` that is false by default. I think that we should keep the pattern for code uniformity. 

I believe there is a story to change this default, but we should probably keep false until that story is done.

Reply:
I did it for similarity with the Portal, where the password is also hidden by default, but yes I'll leave it with `hide-password` for consistency within the CLI then

And yes, we have a story to change other `hide-password` to `show-password`